### PR TITLE
IGNITE-21968: Extend test coverage for SQL E141-07(Basic integrity constraints. Column defaults)

### PR DESCRIPTION
**Pull request by lowka**
Marks tests.

https://issues.apache.org/jira/browse/IGNITE-21968

---

Thank you for submitting the pull request.

To streamline the review process of the patch and ensure better code quality
we ask both an author and a reviewer to verify the following:

### The Review Checklist
- [ ] **Formal criteria:** TC status, codestyle, mandatory documentation. Also make sure to complete the following:  
\- There is a single JIRA ticket related to the pull request.  
\- The web-link to the pull request is attached to the JIRA ticket.  
\- The JIRA ticket has the Patch Available state.  
\- The description of the JIRA ticket explains WHAT was made, WHY and HOW.  
\- The pull request title is treated as the final commit message. The following pattern must be used: IGNITE-XXXX Change summary where XXXX - number of JIRA issue.
- [ ] **Design:** new code conforms with the design principles of the components it is added to.
- [ ] **Patch quality:** patch cannot be split into smaller pieces, its size must be reasonable.
- [ ] **Code quality:** code is clean and readable, necessary developer documentation is added if needed.
- [ ] **Tests code quality:** test set covers positive/negative scenarios, happy/edge cases. Tests are effective in terms of execution time and resources.

### Notes
- [Apache Ignite Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Java+Code+Style+Guide)

### CODE REVIEW

File: modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java [hunk 1]
```diff
@@ -219,6 +219,7 @@ public void explicitColocationColumnsCaseSensitive() {
 
     @Test
     public void literalAsColumDefault() {
+        // // SQL Standard 2016 feature E141-07 - Basic integrity constraints. Column defaults
```
Comment (xtern) [inline modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java:222]:
```suggestion
        // SQL Standard 2016 feature E141-07 - Basic integrity constraints. Column defaults
```

File: modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java [hunk 2]
```diff
@@ -236,6 +237,7 @@ public void literalAsColumDefault() {
     @Test
     @SuppressWarnings("ThrowableNotThrown")
     public void doNotAllowFunctionsInNonPkColumns() {
+        // // SQL Standard 2016 feature E141-07 - Basic integrity constraints. Column defaults
```
Comment (xtern) [inline modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java:240]:
```suggestion
        // SQL Standard 2016 feature E141-07 - Basic integrity constraints. Column defaults
```

File: modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDmlTest.java
Comment (xtern) [inline modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDmlTest.java:1]: Maybe it’s worth adding such a comment into the `testNullDefault` also

File: modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java (hunk 1 above)
Comment (lowka) [reply modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java]: Fixed.

File: modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java (hunk 2 above)
Comment (lowka) [reply modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java]: Fixed.

File: modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDmlTest.java
Comment (lowka) [reply modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDmlTest.java:1]: Done.

**Review by xtern — APPROVED** [review]

**Pull request merged**